playbooks/install_stack.yaml

```diff
@@ -370,9 +374,7 @@
   - name: Install tuned profiles useful for OSP
     yum:
       name:
-      - tuned
-      - tuned-profiles-cpu-partitioning
-      - tuned-profiles-realtime
+      - tuned-profiles-*
```

Comment:
had to change this back because otherwise I get error about tuned-profiles-realtime not being available. 
This should be changed back before merging.

Reply:
yeah I made that change because the packaging on RHEL 8.4 / OSP 16.2 was broken for tuned-*, and it was trying to install mixed versions of tuned. Maybe it's solved now, that RC is out. To be tested again.

Reply:
we should use cpu partitioning anyway, I think you could remove that change.